packages/tlschema/src/shapes/TLGeoShape.ts

```diff
@@ -197,7 +196,10 @@ export const geoShapeMigrations = createShapePropsMigrationSequence({
 		},
 		{
 			id: geoShapeVersions.AddRichText,
-			up: () => {},
+			up: (props) => {
```

Comment:
you can't modify migrations like this, they won't run for recent rooms, you should add a new migration